---
bool success
string error_message
articulation_msgs/ModelMsg model
